contracts/leaser/src/migrate.rs

```diff
@@ -88,15 +119,66 @@ mod test {
         let addr2 = Addr::unchecked("22222");
         let leases = vec![addr1.clone(), addr2.clone()];
 
-        let mut exp = Batch::default();
-        exp.schedule_migrate_wasm_no_reply(&addr1, MigrateMsg {}, new_code)
-            .unwrap();
-        exp.schedule_migrate_wasm_no_reply(&addr2, MigrateMsg {}, new_code)
-            .unwrap();
+        let exp = add_expected(MigrationResult::default(), &addr1, new_code);
+        let mut exp = add_expected(exp, &addr2, new_code);
+        exp.last_instance = Some(addr2);
 
         assert_eq!(
             Ok(exp),
-            super::migrate_leases(leases.into_iter().map(Ok), new_code)
+            super::migrate_leases(leases.into_iter().map(Ok), new_code, 2)
         );
```

Comment:
Expected expressions are second arguments to the macro.
```suggestion
        assert_eq!(
            super::migrate_leases(leases.into_iter().map(Ok), new_code, 2)
            Ok(exp)
        );
```

Reply:
same